src/averageAges.js

```diff
@@ -14,12 +30,13 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const AverageManAge = century
+    ? people.filter(person => Math.ceil(person.died / 100) === century)
+    : people;
+
+  return getAverageAge(getSex(AverageManAge, 'm'));
```

Comment:
Ohh, you use `filter` two times here, let's immediately filter only man ages? In this case, we need to use `filter` only once